tools/devtool

```diff
@@ -556,6 +556,9 @@ cmd_test() {
 
     # If we got to here, we've got all we need to continue.
     say "Kernel version: $(uname -r)"
+    if [ "$(uname --machine)" = "x86_64" ]; then
+        say "Microcode version: $(sudo cat /sys/devices/system/cpu/cpu0/microcode/version)"
```

Comment:
Isn't this included in the lscpu below?

I would be OK removing the lscpu call, as I usually have to scroll over it

Reply:
> Isn't this included in the lscpu below?

I cannot see it in the `lscpu` result.

> I would be OK removing the lscpu call, as I usually have to scroll over it

I'm using the `lscpu` result to know if m5d.metal is skylake or cascade lake.

Reply:
I could swear it was, but you are right. Thanks!